how-to-guides/neptune-docker/scripts/requirements.txt

```diff
@@ -1,4 +1,3 @@
 neptune-client
-psutil
 torch==1.9.0
```

Comment:
great that you dropped psutil -> as we now have it as base dependency :+1:

Reply:
I noticed the change in the dev releases